react-project/src/components/ItemM/index.js

```diff
@@ -0,0 +1,7 @@
+import React from "react"
```

Comment:
Add semicolon after every statement. You can install Prettier extension that does this for you